src/build.py

```diff
@@ -620,17 +625,26 @@ def Fastcomp():
 
 def Emscripten():
   buildbot.Step('emscripten')
-  os.environ['EMSCRIPTEN'] = EMSCRIPTEN_SRC_DIR
-  os.environ['LLVM'] = os.path.join(INSTALL_DIR, 'fastcomp', 'bin')
-  os.environ['V8'] = os.path.join(INSTALL_DIR, 'bin', 'd8')
   em_config = os.path.join(INSTALL_DIR, 'emscripten_config')
+  emscripten_dir = os.path.join(INSTALL_DIR, 'emscripten')
+  os.environ['EM_CONFIG'] = em_config
+  Remove(emscripten_dir)
+  shutil.copytree(EMSCRIPTEN_SRC_DIR,
+                  emscripten_dir,
+                  symlinks=True,
+                  ignore=shutil.ignore_patterns('.git'))
   shutil.copy2(os.path.join(SCRIPT_DIR, 'emscripten_config'),
                os.path.join(em_config))
   try:
     proc.check_call([
-        os.path.join(EMSCRIPTEN_SRC_DIR, 'emcc'),
-        '--em-config', em_config,
+        os.path.join(emscripten_dir, 'emcc'),
```

Comment:
Oh and actually that's kind of a pain to do because shutil.copytree requires that the destination dir not exist. So you basically have to reimplement cp -r.

Which I guess I already did in PNaCl's build system so whatever.